app-crypt/pius/pius-3.0.0.ebuild

```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2018 Gentoo Foundation
```

Comment:
Update this copyright header, `-2020 Gentoo Authors`. You can commit with a tool called **repoman** which will fix most of these common mistakes for you, and even sign-off your commit automatically once you set it up.